Tutorials/seL4Kernel/get-the-tutorials.md

```diff
@@ -0,0 +1,32 @@
+---
+toc: true
+layout: tutorial
+SPDX-License-Identifier: CC-BY-SA-4.0
+SPDX-FileCopyrightText: 2024 seL4 Project a Series of LF Projects, LLC.
+---
+
+# Tutorials
+## Python Dependencies
+Additional python dependencies are required to build [tutorials](ReworkedTutorials). To install you can run:
+```
+pip install --user aenum
+pip install --user pyelftools
```

Comment:
```suggestion
pip3 install --user aenum pyelftools
```
 
These packages should already be included in `camkes-deps` which you'd need at least when you get to the CAmkES parts, but possibly already before. W should make sure that the standard install instructions tell people to do `pip3 install --user camkes-deps`, which should get them everything that is needed.

Reply:
Is it sufficient (and correct) to say

# Getting the tutorials
## Python Dependencies
Additional python and CAmkES dependencies are required to build [tutorials](ReworkedTutorials). To install you can run:
```
pip3 install --user aenum pyelftools camkes-deps

Reply:
Have put a new comment on the new file so it's easier to track. We only need `camkes-deps`, the `aenum` and `pyelftools` are already contained in that.